src/main/java/seedu/apollo/command/module/AddModuleCommand.java

```diff
@@ -228,6 +236,67 @@ public ArrayList<LessonType> getLessonTypes(Module module) {
         return lessonTypes;
     }
 
+    /**
+     * Checks if the lesson clashes with another lesson.
```

Comment:
In your documentation, you can add more details to explain how the function works, ie, that the user will be informed of a clash on the first instance of an overlapping lesson

Reply:
Sounds good! Will do!